### PR TITLE
nvidia-smi: update README.md

### PR DESCRIPTION
**Pull request by vincentkersten**
Save a little time for the next gpu enabler

### CODE REVIEW

**Review by ilyam8 — DISMISSED** [review]

File: collectors/python.d.plugin/nvidia_smi/README.md
Comment (joelhans) [inline collectors/python.d.plugin/nvidia_smi/README.md:20]:
```suggestion
-   You must enable this plugin, as its disabled by default due to minor performance issues:
    ```bash
    cd /etc/netdata   # Replace this path with your Netdata config directory, if different
    sudo ./edit-config python.d.conf
    ```
    Remove the '#' before nvidia_smi so it reads: `nvidia_smi: yes`.
```

**Review by joelhans — CHANGES_REQUESTED** [review]
Thanks for the tweak, @vincentkersten!

Just a bit of formatting tweaks so that the Markdown renders nicely on the generated docs.

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Comment by ilyam8** [comment]
@joelhans please review, can't merge until you approve it

**Review by joelhans — APPROVED** [review]

**Pull request merged**
